meta-ivi/recipes-extended/common-api/capicxx-core-native_3.1.12.4.bb

```diff
@@ -6,4 +6,4 @@ LAUNCHER_LINK = "capicxx-core-gen"
 require capicxx-native.inc
 
 SRC_URI = "https://github.com/GENIVI/${_BPN}-tools/releases/download/${PV}/commonapi-generator.zip"
-SRC_URI[sha256sum] = "979abe3c2ff29e6c5e8a61cdfbd262bf837e6a26201593c5e6e38160978f2a21"
+SRC_URI[sha256sum] = "6de1720a80c6b61c1f20512e03f86c29b21f4df572c89e1f9b2e38923177b3ea"
```

Comment:
This seems to indicate a missing newline.

Reply:
Yes you're right. My fault I will fix it